integreat_cms/api/v3/locations.py

```diff
@@ -71,9 +71,25 @@ def transform_poi_translation(poi_translation: POITranslation) -> dict[str, Any]
 
     poi = poi_translation.poi
 
-    primary_contact = Contact.objects.filter(
-        location=poi, area_of_responsibility=""
-    ).first()
+    contacts = Contact.objects.filter(location=poi).all()
+    contact_data = []
+    for contact in contacts:
+        contact_data.append(
+            {
+                "id": contact.id,
```

Comment:
```suggestion
```

(NOT SURE!) Is there a point to delivering this to the app?

Reply:
Oh, you're right, probably not :)